internal/app/tfsec/gke_abac_enabled_test.go

```diff
@@ -0,0 +1,37 @@
+package tfsec
+
+import (
+	"testing"
+
+	"github.com/liamg/tfsec/internal/app/tfsec/scanner"
+
+	"github.com/liamg/tfsec/internal/app/tfsec/checks"
+)
+
+func Test_GkeAbacEnabled(t *testing.T) {
+
+	var tests = []struct {
+		name                  string
+		source                string
+		mustIncludeResultCode scanner.CheckCode
+		mustExcludeResultCode scanner.CheckCode
+	}{
+		{
+			name: "check google_container_cluster with enable_legacy_abac set to false",
```

Comment:
Small typo here - I think it should be:
```suggestion
			name: "check google_container_cluster with enable_legacy_abac set to true",
```